fetch.bs

```diff
@@ -4153,6 +4164,11 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
    `<code>User-Agent</code>`/<a>default `<code>User-Agent</code>` value</a>
    to <var>httpRequest</var>'s <a for=request>header list</a>.
 
+   <li><p>If <var>httpRequest</var>'s <a for=request>allows stale content flag</a> is set and
+   <var>httpRequest</var>'s <a for=request>header list</a> <a for="header list">contains</a>
+   <code>Cache-Control: Must-Revalidate</code>, then unset <var>httpRequest</var>'s
+   <a for=request>allows stale content flag</a>.
```

Comment:
I'm sorry, I had missed the approach changed substantially (at least I thought there was an enum involved originally). This doesn't quite work. You need to extract the header value and parse/compare it in some manner.

Reply:
I've made it clear that we are looking for a directive in the header. Please let me know if this isn't sufficient. I've looked for other examples of parsing the cache-control header but didn't find any.